docs/package-management.md

```diff
@@ -0,0 +1,19 @@
+# Package Management in the Monorepo
+
+## Using Local Packages
+
+When working on `diracx-web` or other specified projects, related packages are automatically used from their local versions. This ensures that the latest changes are always in effect during development.
```

Comment:
Which are the "specified projects" here?
This part could be expanded a bit may be.

Reply:
Is there any existing official documentation explaining that? You could include a reference pointing to the documentation.